requirements.yml

```diff
@@ -1,5 +1,5 @@
 # From 'openshift-applier'
 - src: https://github.com/redhat-cop/openshift-applier
-  version: v2.1.1
+  version: master
```

Comment:
@ramius345 can you try applier v2.1.2?

Reply:
This needs to stay master until jinja_vars gets put into a release.